src/main/java/com/epam/ta/reportportal/core/filter/predefined/PredefinedFilters.java

```diff
@@ -65,15 +72,37 @@ public Queryable build(String[] params) {
 						.withCondition(new FilterCondition(Operator.OR, Condition.EXISTS, false, "true", CRITERIA_ISSUE_TYPE))
 						.build();
 				}
-			}).build();
+			})
+			.put(USERS, new PredefinedFilterBuilder() {
+				@Override
+				public Queryable build(String[] params) {
```

Comment:
Maybe it's better to add FilterTarget and Set<FilterCondition> to 'build' method arguments and get rid of anonymous class?